.golangci.yml

```diff
@@ -57,16 +61,15 @@ linters:
     - whitespace
 
   # don't enable:
-  # - gochecknoinits
-  # - funlen
-  # - lll
-  # - gomnd
   # - asciicheck
   # - gochecknoglobals
```

Comment:
nit: for alphabetical order please put this package after dupl and funlen

Reply:
🤦  I thought I sorted'em